projects/plugins/jetpack/extensions/blocks/dialogue/dialogue.php

```diff
@@ -213,20 +213,24 @@ function render_block( $attrs, $block_content, $block ) {
 	$participants     = get_participantes_list( $block, $default_participants );
 	$participant_slug = get_participant_slug( $sanitized_attrs, $block, $default_participants );
 	$participant_name = get_participant_name( $participants, $participant_slug, $sanitized_attrs );
-	$css_classname    = Blocks::classes( FEATURE_NAME, $attrs );
+	// Class list includes custom classes defined in block settings.
+	$block_class_list = Blocks::classes( FEATURE_NAME, $attrs );
+	// Only the generated class name for the block, without custom classes.
+	$block_class = explode( ' ', $block_class_list )[0];
```

Comment:
We are relying on that the first class in the string is the block class. It's defined [here](https://github.com/Automattic/jetpack/blob/751542af65c6db0313154970696c9e7f9bf1f3c7/projects/packages/blocks/src/class-blocks.php#L192). It looks a little bit fragile to me. Wondering whether we should define the class name in a hardcoded way.

```suggestion
	$block_class = 'wp-block-jetpack-dialogue';
```

Anyway, it's a minor one. Feel free to ignore it.

Reply:
That's a good point--I think this kind of bug has come up before. It's a problem that deserves further attention to clarify and standardize how we generate the string that ends up being used as a css class prefix.